logos/logos-ui/src/app/features/statistics/statistics.html

```
@@ -172,6 +206,17 @@ <h1 class="page-title">Statistics</h1>
           ariaLabel="Select provider"
           (valueChange)="setSelectedVramProvider($event)"
         />
+        @if (selectedVramProvider() && vramReady() && selectedProviderFreeVramMb() > 0 && !selectedProviderCalibrating()) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)